render-test/android/app/src/main/assets/to_zip.txt

```diff
@@ -11,5 +11,6 @@ mapbox-gl-js/test/integration/geojson/
 mapbox-gl-js/test/integration/sprites/
 vendor/mapbox-gl-styles/styles/
 vendor/mapbox-gl-styles/sprites/
+vendor/mvt-fixtures/
```

Comment:
do you need to explicitly tell what folders to pick? e.g. vendor/mvt-fixtures/real-world?